fleetspeak/src/server/internal/cache/cache.go

```diff
@@ -0,0 +1,114 @@
+package cache
+
+import (
+	"sync"
+	"time"
+
+	"github.com/google/fleetspeak/fleetspeak/src/common"
+	"github.com/google/fleetspeak/fleetspeak/src/server/db"
+)
+
+// Controls cache timing behavior, variable only to support unit tests.
+var (
+	maxAge         = 30 * time.Second
+	expireInterval = 5 * time.Minute
+)
+
+// Client is a cache of recently connected clients.
```

Comment:
The name is pretty confusing. I presume you didn't call this "ClientCache", because the package is already named "cache".
At any rate, "Client" sounds to me like a type that represents a single client. I'd call this e.g. "Clients", "ClientCache", "ClientMapping", what do you think?

Reply:
Right, the package is named cache, as I figure we might put other caches here if we ever need them.

So everybody will see it as cache.Client, which I agree is backwards, but otherwise perfect.

cache.Clients leads to absurdities like cache.NewClients
cache.ClientCache seems overly redundant
cache.ClientMapping makes it sound less like a cache and more like a map.

Maybe cache.ClientData? But it isn't less backwards that cache.Client.

Suggest leaving as-is for now, we can fix up the name later if we find one that actually seems nice. Of course it makes sense that we have to worry about both of the hard problems in CS within the same PR.

Reply:
One being caching and the other naming? :)
NewClients doesn't sound bad to me.
ClientData is also better than Client.
I wouldn't leave this as-is; you're the TL, you decide.

Reply:
https://martinfowler.com/bliki/TwoHardThings.html

Maybe Clients. I'll think about it over lunch, see if any better names come to mind.